frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -448,4 +448,4 @@
   "Operator Backed": "Operator Backed",
   "Browse the catalog to discover and deploy operator managed services": "Browse the catalog to discover and deploy operator managed services",
   "Browse the catalog to discover and install Helm Charts": "Browse the catalog to discover and install Helm Charts"
-}
\ No newline at end of file
+}
```

Comment:
@reginapizza after running `yarn i18n` do not modify this file to add a new line.

Reply:
sorry, must have added that line automatically. I've removed it now.